internal/pyxis/submit.go

```diff
@@ -108,19 +116,22 @@ func (p *pyxisClient) SubmitResults(ctx context.Context, certInput *Certificatio
 		}
 	}
 
-	// Create the test results.
-	testResults := certInput.TestResults
-	testResults.ImageID = certImage.ID
-	testResults, err = p.createTestResults(ctx, testResults)
+	// Update the test results with the certification image id to link the results to the image.
+	updatedTestResults := &TestResults{
+		ID:      testResults.ID,
+		ImageID: certImage.ID,
+	}
```

Comment:
Why create a net-new instance here?

Reply:
This is net-new, since for a PATCH, we do not want/need to send the entire payload on the wire, just the bits we care about.